fe/fe-core/src/main/java/com/starrocks/qe/DefaultCoordinator.java

```diff
@@ -182,7 +184,8 @@ public class DefaultCoordinator extends Coordinator {
     private boolean thriftServerHighLoad;
 
     private Supplier<RuntimeProfile> topProfileSupplier;
-    private final AtomicLong lastRuntimeProfileUpdateTime = new AtomicLong(0L);
+    private Supplier<ExecPlan> execPlanSupplier;
+    private final AtomicLong lastRuntimeProfileUpdateTime = new AtomicLong(System.currentTimeMillis() - 1000);
```

Comment:
why - 1000?

Reply:
Emm, be and fe have its own timer, minus 1000 just to make sure the first report will be well processed. it Looks strange indeed.

Reply:
It's unnecessary and weird. it is removed.